### PR TITLE
Faq

### PR DESCRIPTION
**Pull request by nelsonic**
adds basic outline for readme. #1
adds answers to FAQ #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,89 @@
...
+and not _invest_ in training `new` people.
+
+On the _surface_ it can be _tempting_ to only
+hire "_experienced_" people for technical teams;
+after all someone _else_ has paid for their past "mistakes"
+and they have practiced their craft for "_years_",
+so they are more likely to get the work done faster, right?
+
+
+### Our
```
Comment (iteles) [inline README.md:19]: Missing half a title here

File: README.md [hunk 2]
```diff
@@ -1,2 +1,89 @@
...
+
+At dwyl we hypothesize that it's _more effective_
+to train people _from scratch_
+because of the following reasons:
++ Gives people who are new to the industry a start.
++ Avoids bad habits (_like un-tested or un-documented work!_)
++ Instills the habit of continuous learning _early_
+in the person's journey thus avoiding "_senior syndrome_"
+(_where people feel they already know everything!_)
++ New people tend to have a ["beginners mind"](https://en.wikipedia.org/wiki/Shoshin)
```
Comment (iteles) [inline README.md]: "teachable" might require a bit more context if we're going to leave it in

File: README.md [hunk 3]
```diff
@@ -1,2 +1,89 @@
...
+and are more "_teachable_"
```
Comment (iteles) [inline README.md:30]: "teachable" might require a bit more context if we're going to leave it in

File: README.md [hunk 4]
```diff
@@ -1,2 +1,89 @@
...
++ more likely to pass on their knowledge to teammates (_whereas most "senior" people "hoard" their knowledge our of fear of being replaced!_)
+
+<!--
+> NOTE: help very much wanted re-wording these bullet points!
```
Comment (iteles) [inline README.md:34]: 👍

File: README.md (hunk 3 above)
Comment (nelsonic) [reply README.md:30]: please add if you can.
`else` I will try to add tomorrow. 👍

File: README.md (hunk 1 above)
Comment (nelsonic) [reply README.md:19]: indeed. thanks. 👍

**Review by iteles — APPROVED** [review]
This al looks good, thanks for taking the time to put this together :tada:

**Pull request merged**